src/components/screens/public/ConsentProvider/ConsentProviderController.js

```diff
@@ -0,0 +1,84 @@
+import { VuexModule, Action, Module } from 'vuex-class-modules';
+import createController from '@/controllers/createController';
+import { accountsStore as accountsStoreModule } from '@/store';
+import i18n from '@/locales/i18n';
+
+const OPEN_URL_TIMEOUT_MS = 5000;
+
+@Module({ generateMutationSetters: true })
+class ConsentProviderController extends VuexModule {
+  error = '';
```

Comment:
it's not a good idea to save error in controller. Can we move error to component?

Reply:
have one idea, will try to implement

Reply:
@lamartire @kazurus @ifiri 
updated approach with set error message